packages/react-native-web/src/exports/StyleSheet/initialRules.js

```diff
@@ -13,9 +13,9 @@ const resets = [
   'body{margin:0;}',
   // minimal form pseudo-element reset
   'button::-moz-focus-inner,input::-moz-focus-inner{border:0;padding:0;}',
-  'input::-webkit-inner-spin-button,input::-webkit-outer-spin-button,' +
+  ('input::-webkit-inner-spin-button,input::-webkit-outer-spin-button,' +
```

Comment:
I think we can remove this change

Reply:
Because it's exported, it wants this or we can ignore it. Something with the multi-line thing likely